.github/dependabot.yml

```diff
@@ -249,3 +249,9 @@ updates:
     directory: /packages/pangea-sdk/PangeaCyber.Tests
     schedule:
       interval: daily
+
+  - package-ecosystem: npm
+    directory: /docs
+    versioning-strategy: increase
+    schedule:
+      interval: daily
```

Comment:
what's the strategy around how this gets picked up by docs? is this just a manual rebuild from the docs side?

Reply:
Yes it's currently manual and I wasn't planning on changing that (yet?) with this.

Reply:
cool ty